internal/params/flags.go

```diff
@@ -28,6 +28,7 @@ const (
 	BranchFlag                   = "branch"
 	BranchFlagSh                 = "b"
 	ScanIDFlag                   = "scan-id"
+	ScanIDFlagUsage              = "Cx unique scan-id for the scan created in AST"
```

Comment:
Should we use the Cx prefix? Maybe just "Unique scan-id ..." or "Scan-id ..."

Reply:
it seems is not been used. I will remove it thanks!